svelte/src/routes/v2/types/index.ts

```diff
@@ -48,8 +49,8 @@ export type GraphData = {
 	flattenNodes: ConvertedNode[];
 };
 export type GraphDataEdge = {
-	source: GraphDataNode;
-	target: GraphDataNode;
+	source: string;
```

Comment:
not sure about these two

Reply:
graphData is the one where they are already injected by d3. so the source and the target is reference for the node. So it would be string inside convertedData, and reference inside graphData

Reply:
Yeah, but it's not the case when you call is from the filter function yet, the type should probably be renamed